db/migrate/20220712101341_create_post.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :AuthorId
```

Comment:
- [ ] This column name was supposed to have been specified as a reference to the user table when creating the migration or you could have added the column in a separate command. AuthorId should be a reference or a link to the user's table not a column on its own.  After running the command successfully, you'll find the line
`t.references :AuthorId, foreign_key: { to_table: 'users' }` in your migration file for creating posts table, that is:

```ruby
 create_table :posts do |t|
 ..
 ..
 ..
 ..
    t.references :author, foreign_key: { to_table: 'users' } 
  end
```


- [ ] Kindly do the same with the comments and likes table.